### DISCUSSION TITLE
High latency of in-game tablet driver compared to third party driver

### DISCUSSION THREAD

**Discussion by okolenmi** [ppy/osu#22101] [Q&A]
For a long time I used the in-game driver and thought that the high latency was caused by the tablet itself. But today I tried [OpenTabletDriver](https://opentabletdriver.net/) and it's feels much faster, almost like input from my mouse. 
Now I think, it's possible to reduce latency of in-game driver. **Can this issue be fixed in the near future?**

_Sorry, but I don't know how to compare latency before and after. It's just feels much better._

Edit: Windows 10, XP-Pen Star G-640
Easy way to check this: Enable external OpenTabletDriver and click on Tablet -> Enabled in-game switch to feel difference.

**Comment by peppy** [comment]
Sounds weird. It's the same code base so there should not be any latency. Maybe due to them fixing something recently which hasn't been brought across to our end yet?
